docs/_data/sidebar.yml

```diff
@@ -11,31 +11,29 @@
       url: /articles/getting-started/recommended-extensions
     - title: Settings
       url: /articles/getting-started/settings
-    - title: Prettier Code Formatter
-      url: /articles/getting-started/prettier
     - title: Tips & Tricks
       url: /articles/getting-started/tips-and-tricks
     - title: Java Setup
       url: /articles/getting-started/java-setup
 - title: User Guide
   baseUrl: /articles/user-guide/
   children:
-    - title: Org Development
-      url: /articles/user-guide/org-development-model
-    - title: Package Development
-      url: /articles/user-guide/package-development-model
-    - title: CLI Commands
-      url: /articles/user-guide/cli-commands
-    - title: Source Format
-      url: /articles/user-guide/source-format
-    - title: Default Org
-      url: /articles/user-guide/default-org
+    - title: Development Models
+      url: /articles/user-guide/development-models
+    - title: Deploy on Save
+      url: /articles/user-guide/deploy-on-save
     - title: Org Browser
       url: /articles/user-guide/org-browser
+    - title: Prettier Code Formatter
```

Comment:
You will need to add a URL redirect since the path changed. See: https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/docs/_redirects

Reply:
@ntotten could you please review the redirects update. Thanks.